src/goto-programs/restrict_function_pointers.cpp

```diff
@@ -523,7 +526,9 @@ function_pointer_restrictionst::from_json(const jsont &json)
 
   for(auto const &restriction : to_json_object(json))
   {
-    restrictions.emplace(irep_idt{restriction.first}, [&] {
+    std::string pointer_name =
+      resolve_pointer_name(restriction.first, goto_model);
```

Comment:
Is there a test case that failed before this was added?

Reply:
It was not you who missed something - I had failed to do `git add`! Test is now included.